packages/oidc-auth/src/index.ts

```diff
@@ -308,6 +309,9 @@ const generateAuthorizationRequestUrl = async (
     authorizationRequestUrl.searchParams.set('access_type', 'offline')
     authorizationRequestUrl.searchParams.set('prompt', 'consent')
   }
+  if (env.OIDC_AUDIENCE) {
```

Comment:
fyi @wayofthepie, with the latest changes merged in main you will likely need a tweak to include this new OIDC variable in the config object: https://github.com/honojs/middleware/blob/e678c36193bd10f63448b08b6989873095722f61/packages/oidc-auth/src/index.ts#L84-L96

Reply:
Thanks! I can have a look at pulling in the latest changes soon, and update that.